### PR TITLE
Removing expired FulfillmentMixin  + fixing broken tests

### PR DESCRIPTION
**Pull request by tasawernawaz**
I have removed **FulfillmentMixin** from code and its usages are also updated. I have updated **OrderFulfillView** and **post_checkout_callback** for order fulfilment and creating shipping events.

XCOM-307

Please review this @zubair-arbi @awais786 @aamir-khan @ahsan-ul-haq

### CODE REVIEW

File: ecommerce/extensions/api/v2/tests/views/test_orders.py [hunk 1]
```diff
@@ -97,7 +96,7 @@ def test_super_user(self):
 class OrderFulfillViewTests(UserMixin, TestCase):
     def setUp(self):
         super(OrderFulfillViewTests, self).setUp()
-        ShippingEventType.objects.get_or_create(name=FulfillmentMixin.SHIPPING_EVENT_NAME)
```
Comment (zubair-arbi) [inline ecommerce/extensions/api/v2/tests/views/test_orders.py:100]: Declare `SHIPPING_EVENT_NAME = 'Shipped'` at top of the file or as Class variable according to usage

File: ecommerce/extensions/fulfillment/signals.py [hunk 2]
```diff
@@ -11,7 +12,9 @@ def post_checkout_callback(sender, order=None, **kwargs):  # pylint: disable=unu
...
     # We do not want the false report, and we do not want to omit the api module from coverage reports. This is the
     # "happy" medium.
-    from ecommerce.extensions.fulfillment import api
 
-    # TODO Determine if we need to create ShippingEvents first.
-    api.fulfill_order(order, order.lines.all())
+    order_lines = order.lines.all()
+    line_quantities = [line.quantity for line in order_lines]
+
+    shipping_event, __ = ShippingEventType.objects.get_or_create(name='Shipped')
```
Comment (zubair-arbi) [inline ecommerce/extensions/fulfillment/signals.py]: Please add variable `SHIPPING_EVENT_NAME = 'Shipped'` at top of the file and use it in test files after importing from here.

**Comment by zubair-arbi** [comment]
lgtm :+1:

**Comment by tasawernawaz** [comment]
@jimabramson Please review this.

**Comment by clintonb** [comment]
:+1:

**Comment by jimabramson** [comment]
:+1:

**Pull request merged**